activiti-cloud-api/pom.xml

```diff
@@ -12,7 +12,7 @@
   <name>Activiti Cloud :: Runtime API Parent</name>
   <packaging>pom</packaging>
   <properties>
-    <activiti.version>8.2.0-alpha.12</activiti.version>
+    <activiti.version>0.0.1-PR-4448-1690-SNAPSHOT</activiti.version>
```

Comment:
⏰ Reminder to revert when https://github.com/Activiti/Activiti/pull/4448 is merged